requirements.txt

```diff
@@ -1,4 +1,3 @@
 lcls-krtc
-six
```

Comment:
This must be nice to remove

Reply:
Yeah, a whole dependency for a single standard library function...